### DISCUSSION TITLE
Buying Complete cardreader?

### DISCUSSION THREAD

**Discussion by FfReAkDude** [sanni/cartreader#95] [Need help]
Hello,
Does someone sell These DIY-cardreaders ready to use? I tried soldering an flashing until a certain point but my skills seem to low. If someone sells them, how much would it be?
Thank you!

**Comment by Androxilogin** [comment]
I actually have 14 ready to go. I'd most likely charge around $250 plus shipping. That doesn't include an sd card and is a non battery compartment version.

**Comment by sanni** [comment]
If you already have the parts you can also check if you have a Makerspace/FabLab close by. The Cart Reader is just an Arduino after all so the needed knowledge and skills should be available at such places.
